WS2012R2/lisa/xml/SR-IOV.xml

```diff
@@ -391,6 +392,26 @@
             <timeout>1000</timeout>
         </test>
 
+        <test>
+            <testName>One_vCPU</testName>
+            <testScript>SR-IOV_VerifyVF_basic.sh</testScript>
+            <files>remote-scripts/ica/SR-IOV_VerifyVF_basic.sh,remote-scripts/ica/utils.sh,remote-scripts/ica/SR-IOV_Utils.sh</files> 
+            <setupScript>
+                <file>setupscripts\RevertSnapshot.ps1</file>
+                <file>setupScripts\ChangeCPU.ps1</file>
+                <file>setupscripts\SR-IOV_enable.ps1</file>
+            </setupScript> 
+            <noReboot>False</noReboot>
+            <testParams>
+                <param>NIC=NetworkAdapter,External,SRIOV,001600112800</param>
```

Comment:
do we require to use static MAC?

Reply:
Removing the MAC address would require additional code change & extended testing. I suggest we note it in the Issues tab as an automation improvement to do and work on it separately.